doc/platform_api/CMIS_and_C-CMIS_support_for_ZR.md

```diff
@@ -0,0 +1,1559 @@
+    bias_xp                      = FLOAT                            ; modulator bias xp in percentage
+    bias_yi                      = FLOAT                            ; modulator bias yi in percentage
+    bias_yq                      = FLOAT                            ; modulator bias yq in percentage
+    bias_yp                      = FLOAT                            ; modulator bias yp in percentage
+
+##### 2.1.3 Transceiver DOM Threshold Table #####
+
+    ; Defines Transceiver DOM threshold info for a port
+    key                          = TRANSCEIVER_DOM_THRESHOLD|ifname ; DOM threshold information for module on port
+    ; field                      = value
+    temphighalarm                = FLOAT                            ; temperature high alarm threshold in Celsius
+    temphighwarning              = FLOAT                            ; temperature high warning threshold in Celsius
+    templowalarm                 = FLOAT                            ; temperature low alarm threshold in Celsius
+    templowwarning               = FLOAT                            ; temperature low warning threshold in Celsius
+    vcchighalarm                 = FLOAT                            ; vcc high alarm threshold in V
+    vcchighwarning               = FLOAT                            ; vcc high warning threshold in V
+    vcclowalarm                  = FLOAT                            ; vcc low alarm threshold in V
+    vcclowwarning                = FLOAT                            ; vcc low warning threshold in V
+    txpowerhighalarm             = FLOAT                            ; tx power high alarm threshold in dBm
+    txpowerlowalarm              = FLOAT                            ; tx power low alarm threshold in dBm
+    txpowerhighwarning           = FLOAT                            ; tx power high warning threshold in dBm
+    txpowerlowwarning            = FLOAT                            ; tx power low alarm threshold in dBm
+    rxpowerhighalarm             = FLOAT                            ; rx power high alarm threshold in dBm
+    rxpowerlowalarm              = FLOAT                            ; rx power low alarm threshold in dBm
+    rxpowerhighwarning           = FLOAT                            ; rx power high warning threshold in dBm
+    rxpowerlowwarning            = FLOAT                            ; rx power low warning threshold in dBm
+    txbiashighalarm              = FLOAT                            ; tx bias high alarm threshold in mA
+    txbiaslowalarm               = FLOAT                            ; tx bias low alarm threshold in mA
+    txbiashighwarning            = FLOAT                            ; tx bias high warning threshold in mA
+    txbiaslowwarning             = FLOAT                            ; tx bias low warning threshold in mA
+    lasertemphighalarm           = FLOAT                            ; laser temperature high alarm threshold in Celsius
```

Comment:
Can you please share the offset details for this as well

Reply:
Memory Map info here: https://github.com/sonic-net/sonic-platform-common/blob/86bab38c723eb4ebbfa16feed66344d1b3ffd46e/sonic_platform_base/sonic_xcvr/mem_maps/public/cmis.py#L234

The function to calculate laser temperature here: https://github.com/sonic-net/sonic-platform-common/blob/86bab38c723eb4ebbfa16feed66344d1b3ffd46e/sonic_platform_base/sonic_xcvr/api/public/cmis.py#L818

Table 8-9 in this doc: http://[www.qsfp-dd.com/wp-content/uploads/2021/05/CMIS5p0.pdf](http://www.qsfp-dd.com/wp-content/uploads/2021/05/CMIS5p0.pdf)